docs/source/using/command_line.rst

```diff
@@ -0,0 +1,22 @@
+Using the Command Line
+======================
+
+Both the BDSS and the Workflow Manager can be envoked from the command line.
+
+.. code-block:: console
+
+    # execute the workflow
+    $ edm run -e force-py36 -- force_bdss workflow.json
+
+    # open the Workflow Manager with the workflow loaded
+    $ edm run -e force-py36 -- force_wfmanager workflow.json
+
+    # open the Workflow Manager
+    $ edm run -e force-py36 -- force_wfmanager
+
+Within the force-py36 environment of the Enthought Deployment Manager (EDM), the commands
+are the same but without ``edm run -e`` at the start.
+
+.. code-block:: console
+
+    (force-py36)(edm)$ force_wfmanager
```

Comment:
It's probably clearer to reverse these situations, otherwise the commands get a bit complicated for anyone new to EDM - we also want to encourage users enter the shell environment first for the same reason.

```suggestion
Both the BDSS and the Workflow Manager can be invoked from the command line whilst running a bash shell in the  ``force-py36`` Enthought Deployment Manager (EDM) environment. 

.. code-block:: console

    # enter a shell running the environment
    $ edm shell -e force-py36

    # execute the workflow
    (force-py36)$ force_bdss workflow.json

    # open the Workflow Manager with the workflow loaded
    (force-py36)$ force_wfmanager workflow.json

    # open the Workflow Manager
    (force-py36)$ force_wfmanager

The ``force_bdss`` command initiates the BDSS MCO runner, and therefore must be passed a workflow JSON 
that contains optimization instructions. The ``force_wfmanager`` command initiates the Workflow Manager 
GUI, and therefore can start up with a default empty workflow, since it provides additional UI features to 
create, modify and export workflows. 

EDM also supports running commands from outside a shell, using the ``edm run``command.

.. code-block:: console

    $ edm run -e force-py36 -- force_wfmanager
    
For further assistance on EDM, use the ``edm --help`` tool or visit the 
`latest documentation <https://docs.enthought.com/edm/>`_.
```

Reply:
@sparsonslab - I've updated this commit suggestion to include a description of the `--evaluate` mode